config/language/es/views.view.frontpage.yml

```diff
@@ -21,7 +22,7 @@ display:
             items_per_page_options_all_label: '— Todo(s) —'
             offset_label: Desplazamiento
           tags:
-            previous: '‹ Previous'
+            next: 'Siguiente ›'
```

Comment:
Of course @juampynr we have a deal...As you can imagine this wasn't intentional, Drupals apparently CMI still has some issues. It seems exports strings from Spanish when spanish is negotiated. I'll look into it. In any case, I'm not a big fan of having that one config.tar.gz and will open another issue to discuss the Configuration Managment, which to me seems a bit of a hassle the way it's working right now.